### PR TITLE
Feat/highest scoring visitor kz

### PR DESCRIPTION
**Pull request by zkevkev**
This branch adds a functional #highest_ave_visitor_goals to league. It also adds helper methods: #visitor_goals, #visitor_games, #ave_visitor_goals and #highest_ave_visitor_goals and all associated tests.

### CODE REVIEW

File: lib/stat_tracker.rb
Comment (EricBelongea) [inline lib/stat_tracker.rb:1]: Billy and I added this as well. We might have to deal with a merge conflict later.

File: spec/stat_tracker_spec.rb
Comment (EricBelongea) [inline spec/stat_tracker_spec.rb:1]: For corporate: We talked on video chat about what we want to keep. We will have to refactor all of our stat_tracker_specs in the future.

**Review by EricBelongea — APPROVED** [review]
This looks great. Thanks for knocking them out.

File: spec/league_spec.rb
Comment (billyburch) [inline spec/league_spec.rb:1]: Thanks for adding those tests to cover the bases on your methods.

**Review by billyburch — APPROVED** [review]

**Pull request merged**